### PR TITLE
VIO-1911 Build with podman

### PR DESCRIPTION
**Pull request by rodrigc**

### CODE REVIEW

**Review by MatthewHink — APPROVED** [review]

File: .jenkins [hunk 1]
```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env groovy
 
 // Include this shared CI repository to load script helpers and libraries.
-library identifier: 'vapor@1.0.0-RC13', retriever: modernSCM([
+library identifier: 'vapor@1.20.1', retriever: modernSCM([
```
Comment (MatthewHink) [inline .jenkins:4]: Sorry - I think I broke this one with the emulator changes I made last week. I can cut a branch to try to fix it.

Change library identifier to vapor@1.20.2. add 'skipUnitTest': true, as here and it should work: https://github.com/vapor-ware/synse-snmp-plugin/blob/master/.jenkins

**Comment by MatthewHink** [comment]
There is more broken here than I thought. I can take a crack at this.

**Comment by MatthewHink** [comment]
This PR has the fixes for the latest CI and the docker.io prefix. https://github.com/vapor-ware/synse-snmp-base/pull/11

**Comment by rodrigc** [comment]
https://github.com/vapor-ware/synse-snmp-base/pull/11 is a better fix

**Pull request closed without merging**